flags.go

```diff
@@ -336,15 +336,6 @@ func newApp() (app *cli.App) {
 				Name:  "debug_mutex",
 				Usage: "Print debug messages when a mutex is held too long.",
 			},
-
-			/////////////////////////
-			// Client
-			/////////////////////////
-
-			cli.BoolTFlag{
```

Comment:
Please mark it as do not review if its not ready for review.

Reply:
I added Prince and Ayush to review.